src/libraries/Microsoft.Extensions.Configuration.Xml/tests/XmlConfigurationTest.cs

```diff
@@ -688,6 +688,31 @@ public void ThrowExceptionWhenKeyIsDuplicatedWithDifferentCasing()
             Assert.Equal(expectedMsg, exception.Message);
         }
 
+        [Fact]
+        public void ThrowExceptionWhenDuplicateKeyOfElementContents()
+        {
+            var xml =
+                @"<settings>
+                    <Data>
+                        <DefaultConnection>
+                            <ConnectionString>TestConnectionString</ConnectionString>
+                            <Provider>SqlClient</Provider>
+                        </DefaultConnection>
+                    </Data>
+                    <data name='defaultconnection'>
+                        <ConnectionString>TestConnectionString1</ConnectionString>
+                        <provider>NewProvider</provider>
+                    </data>
+                </settings>";
+            var xmlConfigSrc = new XmlConfigurationProvider(new XmlConfigurationSource());
+            var expectedMsg = SR.Format(SR.Error_KeyIsDuplicated, "data:defaultconnection:ConnectionString",
+                SR.Format(SR.Msg_LineInfo, 9, 43));
+
+            var exception = Assert.Throws<FormatException>(() => xmlConfigSrc.Load(TestStreamHelpers.StringToStream(xml)));
```

Comment:
I don't love all the `var`s here, but I see it's consistent with the rest of the file.